maliput_cargo/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "maliput_cargo"
+version = "0.1.0"
+edition = "2021"
```

Comment:
```suggestion
edition = "2024"
```

Reply:
Check this: https://doc.rust-lang.org/cargo/reference/manifest.html#the-edition-field